docs/reference/tab-widgets/api-call.asciidoc

```diff
@@ -1,5 +1,5 @@
 // tag::cloud[]
-**Use {kib}**
+**Option 1: Use {kib}**
```

Comment:
not sure where this file is being used so I didn't preview it, but nothing looks weird at a glance.